### ISSUE TITLE
React 18 support merged in but not available via npm?

### ISSUE THREAD

**Issue by christianfz** [Detaysoft/react-chat-elements#198]
Hello, thanks for putting this package out and I'm looking forward to trying it!

NPM shows the package's last update was 3 months ago but the release for [v12.0.8](https://github.com/Detaysoft/react-chat-elements/releases/tag/v12.0.8) (React 18 upgrade) was 3 weeks ago.

It looks like the upgrade to React 18 has been merged in but for some reason didn't make it to npm.
![image](https://github.com/Detaysoft/react-chat-elements/assets/130006043/e7b8fc37-bee2-40f1-a52d-2c1ac8fff066)

When I try to install it I get -
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @marageti/z4-lib@1.4.12
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   dev react@"^18.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.2" from react-chat-elements@12.0.8
npm ERR! node_modules/react-chat-elements
npm ERR!   react-chat-elements@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Is this expected? 
Or is there a npm package update already scheduled?

Thanks much!

**Comment by abdurrahmanekr** [comment]
@emregudur could you rebuild and update npm distribution?

**Comment by emregudur** [comment]
It's fixed in 12.0.9 version

**Comment by christianfz** [comment]
Super appreciate the quick response, Thank you!
I temporarily got around it by using:
` "react-chat-elements": "Detaysoft/react-chat-elements"`
in package.json.

I can revert now to use the official npm package. Thanks again! 👍

**Comment by christianfz** [comment]
Verified fixed in v12.0.9.
